engine/eConstr.mli

```diff
@@ -298,7 +300,16 @@ val closedn : Evd.evar_map -> int -> t -> bool
 val closed0 : Evd.evar_map -> t -> bool
 
 val subst_univs_level_constr : Univ.universe_level_subst -> t -> t
-val subst_of_rel_context_instance : rel_context -> t list -> t list
+val subst_instance_context : Univ.Instance.t -> rel_context -> rel_context
+
+val subst_of_rel_context_instance : rel_context -> instance -> substl
+val subst_of_rel_context_instance_list : rel_context -> instance_list -> substl
+
+val liftn_rel_context : int -> int -> rel_context -> rel_context
+val lift_rel_context : int -> rel_context -> rel_context
+val substnl_rel_context : substl -> int -> rel_context -> rel_context
+val substl_rel_context : substl -> rel_context -> rel_context
+val smash_rel_context : rel_context -> rel_context
```

Comment:
can you add some doc here?

Reply:
We could either redirect to the doc of the functions of same name in `Vars` or copy the doc of those functions. I think I have a preference for the former.

Reply:
I could not find the doc for this last api.

Reply:
The doc for these functions is in [vars.mli](https://github.com/coq/coq/blob/b5f7c11c2cd1d83a310b82a56095d3cf6829cd6f/kernel/vars.mli).

While we are at it, would you like to assign?